recipes/haproxy_phpbb.rb

```diff
@@ -48,14 +48,12 @@
   # Authentication isn't actually used with IPv6 (see https://tools.ietf.org/html/rfc5798#section-9)
   # however, keepalived cookbook requires this to be defined
   authentication auth_type: 'PASS', auth_pass: secrets['auth_pass']
-  virtual_ipaddress %w() # TODO: IPv6 for lb.phpbb.com
+  virtual_ipaddress %w(2605:bc80:3010:103::8cd3:ff4/64)
   notifies :reload, 'service[keepalived]'
-  action :nothing # TODO: IPv6 for lb.phpbb.com
```

Comment:
Shouldn't this have ``notifies :reload, 'service[keepalived]'`` ?

Reply:
I believe it already does.

Reply:
Oops, you're right!